community/kernel/src/main/java/org/neo4j/kernel/api/DataRead.java

```diff
@@ -33,10 +33,9 @@
 import org.neo4j.kernel.api.properties.Property;
 import org.neo4j.kernel.impl.api.RelationshipVisitor;
 import org.neo4j.kernel.impl.api.store.RelationshipIterator;
-import org.neo4j.kernel.impl.util.register.NeoRegister;
-import org.neo4j.register.Register;
 
 interface DataRead
+        extends DataReadCursors
```

Comment:
Why is this needed? I think it is cleaner to make `ReadOperations` extends both `DataRead` and `DataReadCursors`